.github/workflows/update-electron.yaml

```diff
@@ -22,6 +22,7 @@ jobs:
         with:
           # don't checkout a detatched HEAD
           ref: ${{ github.head_ref }}
+          token: ${{ steps.app-token.outputs.token }}
```

Comment:
The changes to this workflow are an attempt to disassociate @himanshusinghs from the generated commit.